### ISSUE TITLE
[1.10.2] server crash when changing/loading dimensions

### ISSUE THREAD

**Issue by hugeblank** [SpongePowered/SpongeForge#1118]
Issue:
when moving between dimensions, in my case the nether, everything loads for a second (you can see the chunk you are in consists of netherrack), and then the server crashes. I'm pretty sure this is sponge that is causing the issue, because when I looked at the log, it appeared to be some sort of world generation issue, and sponge is the only mod that I know of that handles world generation in the nether. None of the other mods (to my knowledge) handle Nether terrain generation.

To recreate:
1: Create any old Multiplayer world
2: create and step through a nether portal
3: crash
4: reload server
5: log in
6: ...crash
7: repeat steps 4-6 until thoroughly enraged.

EDIT: forgot to add log :P
[crash-2016-12-17_20.52.01-server.txt](https://github.com/SpongePowered/SpongeForge/files/659097/crash-2016-12-17_20.52.01-server.txt)

**Comment by hugeblank** [comment]
I recommend that this issue get fixed ASAP, seeing as it's game breaking. If someone would please at least comment saying that they have read this, it would be amazing.

**Comment by hugeblank** [comment]
Fixed in update 2005. Thank you SO MUCH!

**Comment by hugeblank** [comment]
Thought 2005 fixed it...nevermind :/

**Comment by hugeblank** [comment]
Okay. New results: 
It works! Kinda.
Worlds that previously were corrupted need to be reset. Was that an issue for me? No. From the start I had been having issues with the nether, caused by sponge. So it was never really used. I'd suggest pushing an update that de-corrupts previous worlds, for convenience of other servers.

Unrelated:
I was about to report an issue to you guys for 'sponge not generating structures' I spent like an hour testing different locations where I could find villages, nether fortresses and the like... and then I checked my server.properties. I had generate structures set to false for some random reason.
I think my stupidity is worth a laugh or two.
